src/amo/sagas/collections.js

```diff
@@ -36,12 +36,17 @@ import type {
 } from 'amo/api/collections';
 import type {
   AddAddonToCollectionAction,
+  CollectionAddonsListResponse,
   FetchCurrentCollectionAction,
   FetchCurrentCollectionPageAction,
   FetchUserCollectionsAction,
   UpdateCollectionAction,
 } from 'amo/reducers/collections';
 
+function emptyAddons(): CollectionAddonsListResponse {
+  return { count: 0, next: '', previous: '', results: [] };
```

Comment:
I'm not particularly happy about this, but Flow types plus the automated test should help us keep it in sync if the return values from the API method change. I suppose ideally there'd be something in our API code which could deliver a standard `ListResponse`, but I don't think such a thing exists, and even if it did we'd still need to manually keep it in sync.